examples/editor/src/main.tsx

```diff
@@ -1,14 +1,24 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
 import "./index.css";
+import styles from "./App.module.css";
+import { mountBlockNoteEditor } from "../../../packages/core/src/BlockNoteEditor";
+import { BubbleMenuFactory } from "../../../packages/react/src/BubbleMenu/BubbleMenuFactory";
 
-window.React = React;
-window.ReactDOM = ReactDOM;
+// type WindowWithProseMirror = Window &
+//   typeof globalThis & { ProseMirror: Editor };
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
-);
+mountBlockNoteEditor({
```

Comment:
If we want this example, to be a "react" example, then I think the previous code, with `<App />` and `useEditor` and `<EditorContent>` is better

Reply:
Would it would make sense to have 2 examples, one for vanilla JS and one for React?